src/index.js

```diff
@@ -1,11 +1,47 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, compose } from 'redux';
 import './stylesheet/index.css';
 import App from './components/App';
+import booksReducer from './reducers/book';
+
+const randomID = () => Math.floor(Math.random() * 10);
+
+const initialState = {
+  books: [
+    {
+      ID: randomID(),
+      title: 'House of Sticks',
+      category: 'Biography',
+    },
+    {
+      ID: randomID(),
+      title: 'The Hidden Knife',
+      category: 'Kids',
+    },
+    {
+      ID: randomID(),
+      title: 'Survive The Night',
+      category: 'Horror',
+    },
+  ],
+};
+
+const store = createStore(
+  booksReducer,
+  initialState,
+  compose(
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

Reply:
This case, in particular, was the Redux debugger and I believe it's made that way on purpose :sweat_smile:, but it's fixed now :+1: